test/unit/fetcher.js

```diff
@@ -0,0 +1,165 @@
+import assert from 'assert';
+import { HttpsProxyAgent } from 'https-proxy-agent';
+import Fetcher from 'Provider/Fetcher';
+import { DataTexture, Texture } from 'three';
+
+describe('Fetcher', function () {
+    const networkOptions = process.env.HTTPS_PROXY ? { agent: new HttpsProxyAgent(process.env.HTTPS_PROXY) } : {};
+    describe('text', function () {
+        const url = 'https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/' +
+        'altitude-conversion-grids/EGM2008.gdf';
```

Comment:
Could define a `baseUrl` like you did elsewhere, to use here, and in l.24, l.45 etc..